cdk/package.json

```diff
@@ -33,9 +32,12 @@
 		"typescript": "~5.3.3"
 	},
 	"dependencies": {
-		"@guardian/cdk": "53.1.0",
 		"npm": "^10.4.0",
 		"source-map-support": "^0.5.20"
 	},
-	"prettier": "@guardian/prettier"
+	"resolutions": {
+		"wrap-ansi": "7.0.0"
+	},
+	"prettier": "@guardian/prettier",
+	"packageManager": "yarn@1.22.5"
```

Comment:
Is there a specific reason the `packageManager` is needed here?

Reply:
As if a user has a different/newer version it could cause issues. Maybe define it as a range?

Reply:
Just removed this.

Reply:
thanks